flight/UAVTalk/uavtalk.c

```diff
@@ -391,7 +377,7 @@ UAVTalkRxState UAVTalkProcessInputStreamQuiet(UAVTalkConnection connectionHandle
 		}
 
 		// Check the lengths match
-		if ((iproc->rxPacketLength + iproc->instanceLength + iproc->timestampLength + iproc->length) != iproc->packet_size) { // packet error - mismatched packet size
+		if ((iproc->rxPacketLength + iproc->instanceLength + iproc->length) != iproc->packet_size) { // packet error - mismatched packet size
```

Comment:
Need to keep this so we can still receive timestamped packets okay (even if we just discard the bytes further down). Either that or we need to bump protocol version because removing this would break BC (and our current public protocol definition, would need to update that too) (I really don't think we should do that).